Sources/Sentry/include/SentrySDK+Private.h

```diff
@@ -51,6 +55,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (void)captureEnvelope:(SentryEnvelope *)envelope;
 
++ (void)captureFeedback:(SentryFeedback *)feedback;
```

Comment:
`m`: I'm a bit confused why we have this here in `SentrySDK+Private.h` and in `SentrySDK.h`.

Reply:
probably just me shuffling things around to get them working. will make sure to clean this up 👍🏻